pages/alerts.rst

```diff
@@ -135,7 +135,10 @@ Event Data
         - ``priority`` (long) - The event's priority value.
         - ``fields`` (Map<String, String>) - The custom fields attached to the event.
 
+.. _Backlog:
+
 Backlog
+-------
```

Comment:
Now that you're not linking to this from elsewhere in the documentation, I think you can get rid of the anchor and section heading.

```suggestion
Backlog
```